strobogrammatic.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+class solution:
+	def is_strobogrammatic(self, num):
+		turn_map = {'9':'6', '6': '9', '1':'1', '8':'8'}
```

Comment:
This should be an private class member. 
And also make it constant (e.g. _TURN_MAP).